tracing/suspend_resume_time.bt

```diff
@@ -0,0 +1,24 @@
+// This tracks the time it takes for runtime and system suspend/resume
```

Comment:
commit message: you need to have some sort of prefix, then the action, e.g.

tracing: add suspend and resume time tracking with bptrace

Also add some of the explanations in the commit message itself, not the PR.

Reply:
Good point, just updated.